scripts/settings_speedometer.js

```diff
@@ -483,8 +483,11 @@ class RangeColorProfileObject {
 	}
 
 	createDisplayPanels(profileKV) {
-		Object.keys(this.displayObjectList)?.forEach((id) => this.displayObjectList[id]?.destroy());
+		const displayObjects = Object.keys(this.displayObjectList);
+		if (displayObjects) for (const id of displayObjects) this.displayObjectList[id]?.destroy();
```

Comment:
Use null coalesce here like you do in a later commit.

Reply:
Done this and below in last push